index.html

```diff
@@ -41,14 +69,15 @@
 <section id="abstract">
 
 This finding examines the sustainability of bundling
-and double-keyed caching in web development.
+and storage partitioning in web development.
```

Comment:
Storage partitioning is one aspect of double-keyed caching, see for example "Partitioning Network Cache"

Reply:
The network cache is a storage mechanism :)